packages/flutter/lib/src/widgets/sliver_persistent_header.dart

```diff
@@ -188,6 +188,7 @@ class _SliverPersistentHeaderElement extends RenderObjectElement {
 
   @override
   void performRebuild() {
+    super.performRebuild();
```

Comment:
@HansMuller can you check if this is right? I'm surprised it didn't need to call super since the element's dirty bit is never removed during rebuild.